packages/messaging/src/controllers/base-controller.ts

```diff
@@ -70,14 +70,16 @@ export abstract class BaseController
     this.tokenDetailsModel = new TokenDetailsModel(services);
   }
 
-  async getToken(): Promise<string | null> {
```

Comment:
Do we have release notes where we can tell callers they can remove their null checks?

Reply:
Yup.